docs/pages/klistwithoverflow.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+
+  <DocsPageTemplate apiDocs>
+    <DocsPageSection title="Overview" anchor="#overview">
+      <p>
+        The <code>KListWithOverflow</code> component is useful when we want a horizontal list that is responsive and adaptable to the container size. The list only shows the items that fit on the container element. When items exceed the available space, the component seamlessly integrates a "see more" button to show additional items.
+      </p>
+      <p>
+        This component is renderless, so it needs an array of items, slots to render the visible items and separators on the list, and a slot to render the "see more" button.
+      </p>
+    </DocsPageSection>
+    <DocsPageSection title="Example" anchor="#example">
```

Comment:
Could you add code previews via `DocsShowCode` for both the basic usage example as well the dividers example? This component has a bit complex API so I'd be good to demonstrate how to use it. Examples would ideally have both template part and a script part with items in `data()`. Also, I'd like to ask you don't copy `getItems` to `DocsShowCode` but rather demonstrate with objects `{ label, icon }` literally so the usage is apparent at first glance.